browser-test/src/program_admin/program_admin_program_list.test.ts

```diff
@@ -0,0 +1,59 @@
+import {test} from '../support/civiform_fixtures'
+import {
+  enableFeatureFlag,
+  loginAsAdmin,
+  loginAsProgramAdmin,
+  logout,
+  validateScreenshot,
+} from '../support'
+import {ProgramVisibility} from '../support/admin_programs'
+
+test.describe('Program admin program list', () => {
+  test('shows all the programs that are active', async ({
+    page,
+    adminPrograms,
+  }) => {
+    await test.step('log in as a CiviForm admin and publish multiple programs', async () => {
+      await loginAsAdmin(page)
+      await adminPrograms.addProgram('Program Name One')
+      await adminPrograms.addProgram('Program Name Two')
+      await adminPrograms.addProgram('Program Name Three')
+      await adminPrograms.publishAllDrafts()
+      await logout(page)
+    })
+
+    await test.step('log in as a program admin and view the program list, verify that all active programs are shown', async () => {
+      await loginAsProgramAdmin(page)
```

Comment:
**suggestion, blocking**: It would be good to also check the page directly for the expected programs, using something like
- https://github.com/civiform/civiform/blob/60bea30f2ffa29e05716b35418a66c09d07f8ca0/browser-test/src/support/admin_programs.ts#L82
- or https://github.com/civiform/civiform/blob/60bea30f2ffa29e05716b35418a66c09d07f8ca0/browser-test/src/admin/admin_program_list.test.ts#L147

the former doesn't seem to be used, and the latter might need to be moved into support/admin_programs so you can use it, so I'm not sure which will be easier.

(here and the other test)

Reply:
I think both of the functions are meant to check the program dashboard for civiform admin but not program admin. you can see it from https://github.com/civiform/civiform/blob/60bea30f2ffa29e05716b35418a66c09d07f8ca0/browser-test/src/support/admin_programs.ts#L83
and 
https://github.com/civiform/civiform/blob/60bea30f2ffa29e05716b35418a66c09d07f8ca0/browser-test/src/admin/admin_program_list.test.ts#L154

I can in the future create a folder under support for program admins. Adding it to issue #7209 to track this.

Reply:
ahh good point.

What about just a `await expect(page.getByText('Program Name One')).toHaveCount(1)`? That way there's a programatic check for the program, not just the screenshot.

(Relying solely on the screenshot could result in the test being updated incorrectly, if someone just accepts all the screenshot diffs without noticing)

Reply:
good point! added `await expect(programName).toHaveCount(1)` to check the programs shown on the dashboard